pkg/nsx/services/vpc/vpc.go

```diff
@@ -568,6 +568,15 @@ func (s *VPCService) CreateOrUpdateVPC(obj *v1alpha1.NetworkInfo) (*model.Vpc, *
 		return existingVPC[0], &nc, nil
 	}
 
+	if nc.VPCConnectivityProfile != "" {
+		vpcConnectivityProfile, err := s.NSXClient.VpcConnectivityProfilesClient.Get(nc.Org, nc.NsxtProject, nc.VPCConnectivityProfile)
+		if err != nil {
+			log.Error(err, "vpcConnectivityProfile does not exists", "Project", nc.NsxtProject, "VPCConnectivityProfile", nc.VPCConnectivityProfile)
+			return nil, nil, err
+		}
+		createdVpc.ExternalIpv4Blocks = vpcConnectivityProfile.ExternalIpBlocks
```

Comment:
external_ipv4_blocks will not be allowed with vpc_connectivity_profile or  vpc_service_profile。
Can you create VPC using this body?